### PR TITLE
added sign in action and sign out

### PR DESCRIPTION
**Pull request by nooralnabi**

### CODE REVIEW

File: app/controllers/businesses/registrations_controller.rb [hunk 1]
```diff
@@ -9,14 +9,14 @@ def respond_with(resource, opts = {})
       resource.errors.each do |attr, messages|
         errors_messages << {field_name:attr,details: messages}
       end
-      render json: {status: false , errors: errors_messages }
+      render json: {status: false , errors: errors_messages }, status: :unprocessable_entity
```
Comment (ssharba) [inline app/controllers/businesses/registrations_controller.rb]: visit customer api and copy new exception logic for application controller

Comment (nooralnabi) [reply app/controllers/businesses/registrations_controller.rb]: @ssharba you mean this method error_occurred(exception) in the application controller

File: config/locales/ar.yml [hunk 2]
```diff
@@ -1188,6 +1188,8 @@ ar:
           business_has_profile: هذا العميل يمتلك بروفايل
           Thank_you_for_verifying: شكرًا لك على التحقق من حسابك
           verification_code_sent_successfully: تم إرسال رمز التحقق بنجاح
+          business_inactive_account: "لا يمكنك تسجيل الدخول حتى يقوم المشرف بتنشيط حسابك"
```
Comment (ssharba) [inline config/locales/ar.yml]: @nooralnabi let's use new locale structure please

Comment (nooralnabi) [reply config/locales/ar.yml]: @ssharba I used the new structure with my next PR please marge that

**Pull request merged**